submissions/OlStaniev/DOM/index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./css/style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700;900&display=swap" rel="stylesheet">
+    <title>Star Wars Saga</title>
+</head>
+
+<body>
+    <header class="header">
+        <img src="./img/star-wars.svg" alt="star wars logo" class="header__logo">
+        <button class="header__burgerMenu"></button>
+    </header>
+    <main>
+        <nav class="nav">
+            <ul class="nav__list">
+            </ul>
+        </nav>
+        <div class="introText">
+            <div class="introText__wrapper">
+            </div>
+        </div>
+    </main>
+</body>
+
+<script src="./script/script.js" type="module"></script>
+<script src="./script/text-episode.js" type="module"></script>
+
```

Comment:
Check the page in the [validator](https://validator.w3.org/).
A tag `<script>` must be into the tag `<body>`.